packages/contract-types/package.json

```diff
@@ -2,17 +2,18 @@
   "name": "daimo-contract-types",
   "version": "1.0.0",
   "description": "Typescript types and ABIs for daimo contracts",
-  "main": "src/index.ts",
+  "types": "./dist/contract-types/src/index.d.ts",
+  "main": "./dist/contract-types/src/index.js",
```

Comment:
was something breaking linking to `src/index.ts`?

Building typescript is generally necessary when publishing a library to NPM, but if we're only using it locally from another typescript project, I'm not sure

Reply:
yeah https://github.com/daimo-eth/daimo/pull/25#discussion_r1225900978 requires us to `npm pack` these which only picks from the built files.